packages/main/cypress/specs/Icon.cy.tsx

```diff
@@ -0,0 +1,268 @@
+import Icon from "../../src/Icon.js";
+import Input from "../../src/Input.js";
+import Label from "../../src/Label.js";
+import "@ui5/webcomponents-icons/dist/add-equipment.js";
+import "@ui5/webcomponents-icons/dist/error.js";
+import "@ui5/webcomponents-icons/dist/add.js";
+import "@ui5/webcomponents-icons/dist/add-equipment.js";
+import testAssets from "../../src/bundle.esm.js";
+import { setTheme } from "@ui5/webcomponents-base/dist/config/Theme.js";
+
+describe("Icon general interaction", () => {
+    it("Tests icon rendering", () => {
+        cy.mount(
+            <>
+                <Icon name="add-equipment" mode="Interactive" />
+                <Icon name="save" showTooltip />
+            </>
+        );
+
+        cy.get("[ui5-icon][name='add-equipment'][mode='Interactive']")
+            .shadow()
+            .find(".ui5-icon-root")
+            .should("exist");
+
+        cy.get("[ui5-icon][show-tooltip]").should("have.prop", "showTooltip", true);
+    });
+
```

Comment:
Both cover rendering and tooltip presence, but WDIO explicitly validates tooltip text, while Cypress only checks the property.